src/main.ts

```diff
@@ -63,17 +63,6 @@ export async function _main(): Promise<void> {
 			}
 		});
 
-		// Shout out our source code.
-		client.user.setActivity({
-			// ActivityType.Playing looks like crap, but it's the only way
-			// to show a custom multiline string on the bot's user profile.
-			// I'd put the URL in "About Me", but Discord seems to delete
-			// those sometimes, and I'd like to stay on Discord's good side.
-			type: ActivityType.Playing,
-			name: 'Source: github.com/BYU-CS-Discord/CSBot',
-			url: 'https://github.com/BYU-CS-Discord/CSBot',
-		});
-
```

Comment:
What should we replace this with? Preferably the bot should have _something_ for its activity.

With my bots, I like to have their activities direct users to use the /help command. Thoughts?

Reply:
No idea

Reply:
"Playing /help for info" is fine. I think people get it and its what almost every bot uses unless its something funny.

Reply:
Added